docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3658,12 +3658,10 @@ IS_IPV4( expression )
 * `_expression_`
 +
 specifies an expression that determines the values to include in the
-computation of the minimum. The _expression_ cannot contain an aggregate
-function or a subquery. The DISTINCT clause specifies that the MIN
-function operates on distinct values from the one-column table derived
-from the evaluation of _expression_. All nulls are eliminated before
-the function is applied to the set of values. If the result table is
-empty, IS_IPV4 returns NULL.
+validation of the IP address. The _expression_ cannot contain an aggregate
+function or a subquery. All nulls are eliminated before the function is
+applied to the set of values. If the result table is empty, IS_IPV4 

```

Comment:
Thanks Dave for careful and patient review, this is really helpful. I will update the words.
For the is_ipv4(max(b)) assertion failure, I will do some debug.